packages/pyolite-kernel/src/worker.ts

```diff
@@ -98,87 +106,136 @@ async function sendComm(
  * @param content The incoming message with the code to execute.
  */
 async function execute(content: any) {
-  const stdoutCallback = (stdout: string): void => {
+  const publishExecutionResult = (
+    prompt_count: any,
+    data: any,
+    metadata: any
+  ): void => {
+    const bundle = {
+      execution_count: formatResult(prompt_count),
```

Comment:
Does `prompt_count` need to be passed through `formatResult`, or could it be used directly if it already comes as a `number`?

Reply:
Yes, it can be passed directly, fixed it.